src/main/scala/io/sdkman/changelogs/PierrotMigrations.scala

```diff
@@ -0,0 +1,51 @@
+package io.sdkman.changelogs
+
+import com.github.mongobee.changeset.{ChangeLog, ChangeSet}
+import com.mongodb.client.MongoDatabase
+
+@ChangeLog(order = "063")
+class PierrotMigrations {
+  @ChangeSet(
+    order = "001",
+    id = "001_add_pierrot_candidate",
+    author = "musketyr"
+  )
+  def migration001(implicit db: MongoDatabase) =
+    Candidate(
+      candidate = "pierrot",
+      name = "Pierrot",
+      description =
+        "Pierrot helps you to manage many GitHub repositories with a single command.",
+      websiteUrl = "https://agorapulse.github.io/pierrot/",
+      distribution = "PLATFORM_SPECIFIC"
+    ).insert()
+
+  @ChangeSet(
+    order = "002",
+    id = "002-add_pierrot_0_9_8",
+    author = "musketyr"
+  )
+  def migration002(implicit db: MongoDatabase) = {
```

Comment:
We probably don't want versions here, because you will be doing this with the API.

Reply:
can we do just the initial here to have something to start with before I setup the integration?

Reply:
SDKMAN will report it on the list command and on the website as "coming soon", but I'm happy to go ahead with these versions if you want.

Reply:
yes, please :-) it might take some time to implement the integration so I would like to have some starter version